pkg/secrets/k8s/k8s_secrets_test.go

```diff
@@ -41,6 +41,26 @@ func TestKubernetesSecrets(t *testing.T) {
 			})
 		})
 
+		Convey("Given a map of data entries and a secret with backslashes", func() {
+			m := make(map[string][]byte)
+			// need to simulate a conjur secret, with 1 backslash, so we use unicode to simulate this
+			m["user"] = []byte("super\u005csecret");
+			DataEntry, err := generateStringDataEntry(m);
+
+			Convey("Finishes without raising an error", func() {
+				So(err, ShouldEqual, nil)
+			})
+
+			Convey("Returns proper password with escaped characters", func() {
+				// before sending secret to k8s there are two backslashes
```

Comment:
secrets? a secret?

Reply:
improve your english in this paragraph please :)

Reply:
in terms of english, it is correct. It is just in the passive voice. I will update this to be active